rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml

```
@@ -26,11 +26,13 @@ risk_score = 73
 rule_id = "9092cd6c-650f-4fa3-8a8a-28256c7489c9"
 severity = "high"
 tags = ["Elastic", "Host", "macOS", "Threat Detection", "Credential Access"]
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:security and process.args:("find-generic-password" or "find-internet-password")
+process where event.type == "start" and
+ process.name:"security" and process.args : "-wa" and process.args:("find-generic-password", "find-internet-password") and
+ process.args : ("Chrome*", "Chromium", "Opera", "Safari*", "Brave", "Microsoft Edge", "Edge", "Firefox*") and
+ not process.parent.executable : "/Applications/Keeper Password Manager.app/Contents/Frameworks/Keeper Password Manager Helper*/Contents/MacOS/Keeper Password Manager Helper*"
```

nit for consistency:

```suggestion
process where event.type == "start" and
 process.name : "security" and process.args : "-wa" and process.args : ("find-generic-password", "find-internet-password") and
 process.args : ("Chrome*", "Chromium", "Opera", "Safari*", "Brave", "Microsoft Edge", "Edge", "Firefox*") and
 not process.parent.executable : "/Applications/Keeper Password Manager.app/Contents/Frameworks/Keeper Password Manager Helper*/Contents/MacOS/Keeper Password Manager Helper*"
```